### PR TITLE
feat: add BulletedListItem

### PR DESCRIPTION
**Pull request by anonymousRecords**
# Description of Changes

Here is reference: #25

---

## support 'Bulleted List Item' block

- add custom solution for rendering 'Bulleted List Item' block.

## Review point
- Is the component structure clear and easy to understand?

## To reproduce

- npm run story:start
- click 'your created block' on the Storybook left side panel

## Screenshot

image

## Review Guide

Reviews are conducted based on priority levels, such as p0, p1, p2, p3, p4, and p5.  
p0 ~ p2: If the author decides not to reflect a review for p0 to p2, it signals that a proper discussion with the reviewer is  
necessary. It is expected that the review will be resolved either through incorporating the feedback or through further discussion.  
p3: indicates that the reviewer has identified a significant issue, but either lacks a clear solution or the comment lacks sufficient context. Further explanation or additional discussion on the reviewer's concerns is needed.
p4, p5: p4 and p5 suggest low priority, and if the author does not deem them important, these comments can be disregarded.

### CODE REVIEW

File: packages/react-notion-custom/src/lib/index.css [hunk 1]
```diff
@@ -564,6 +564,31 @@
   padding: 3px 2px;
 }
 
+.notion-list-bulleted {
+  list-style-type: none;
+  padding-left: 0;
+}
+
```
Comment (Moon-DaeSeung) [inline packages/react-notion-custom/src/lib/index.css:571]: p2: Please place the CSS at the very bottom to minimize merge conflicts.

**Comment by Moon-DaeSeung** [comment]
p1: we have a pr template

could you edit your pr description regarding this: https://github.com/meursyphus/react-notion-custom/pull/28

File: packages/react-notion-custom/src/lib/index.css (hunk 1 above)
Comment (anonymousRecords) [reply packages/react-notion-custom/src/lib/index.css]: > p2: Please place the CSS at the very bottom to minimize merge conflicts.

I've moved the CSS to the very bottom of the file as requested.

**Comment by anonymousRecords** [comment]
@Moon-DaeSeung 
> p1: we have a pr template
>
> could you edit your pr description regarding this: https://github.com/meursyphus/react-notion-custom/pull/28

I've updated the PR description to follow the template provided in #28.

**Comment by Moon-DaeSeung** [comment]
sorry for making your pr in merge conflict

could you resolve this?

**Comment by anonymousRecords** [comment]
@Moon-DaeSeung 
> sorry for making your pr in merge conflict
>
> could you resolve this?

Done

**Pull request merged**